tests/BuildScriptGenerator.Tests/DefaultBuildScriptGeneratorTest.cs

```diff
@@ -372,74 +372,6 @@ public void TryGenerateScript_CallsDetector_IfMultiPlatformIsOn_AndLangProvided(
             Assert.True(detector2.DetectInvoked);
         }
 
-        [Fact]
-        public void UsesMaxSatisfyingVersion_WhenOnlyMajorVersion_OfLanguageIsSpecified()
```

Comment:
q: why are we getting rid of these tests?

Reply:
As discussed offline, removing these tests are they not a common scenario (i.e where we would have multiple instances of platform for the same language name)